modules/argocd/argocd-application/helm/templates/application.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: argoproj.io/v1alpha1
 kind: Application
 metadata:
   name: {{ .Values.name }}
-  namespace: {{ .Values.namespace }}
```

Comment:
Why is the namespace here being removed?

Reply:
When using helm charts the namespace should be specified in the templates, the namespace to install the chart content in this case Application is dictated by the helm_release or helm cli in this case is specified at the `applications` allowing the Application to be installed in `argocd` namespace when using ArgoCD as Stand-alone instance and in case of multi-cluster centralized ArgoCD Instance then installed in the namespace representing the remote cluster, in our example `cluster-dev`, `cluster-test` and `cluster-prod` namespaces in the hub cluster